aea/cli/generate_key.py

```diff
@@ -68,8 +75,64 @@ def generate_key(
     add_key: bool = False,
     connection: bool = False,
     extra_entropy: Union[str, bytes, int] = "",
+    n: Optional[int] = None,
 ) -> None:
     """Generate a private key and place it in a file."""
+    if n is None:
+        _generate_one(
+            click_context=click_context,
+            type_=type_,
+            file=file,
+            password=password,
+            add_key=add_key,
+            connection=connection,
+            extra_entropy=extra_entropy,
+        )
+        return
+
+    _generate_many(
+        n=n,
+        type_=type_,
+        password=password,
+        extra_entropy=extra_entropy,
+        file=file,
+    )
+
+
+def _generate_many(
+    n: int,
+    type_: str,
+    password: Optional[str] = None,
+    extra_entropy: Union[str, bytes, int] = "",
+    file: Optional[str] = None,
+) -> None:
+    """Generate n keys."""
+
+    key_pairs = []
+    for _ in range(n):
+        crypto = make_crypto(type_, extra_entropy=extra_entropy)
+        priv_key = (
+            crypto.encrypt(password=password)
+            if password is not None
+            else crypto.private_key
+        )
+        key_pairs.append({ADDRESS: crypto.address, PRIVATE_KEY: priv_key})
```

Comment:
The keys will be generated and dumped as a json list in following format

```json
[
  {
    "address": "address_string",
    "private_key": "private_key_string_or_encrypted_json_string"
  }
]
```

Reply:
This should be a utility method in `aea.crypto.helpers` similarly to create_private_key. Otherwise we have similar logic in different places.